index.js

```diff
@@ -249,7 +249,12 @@ exports.getHashDigest = function getHashDigest(buffer, hashType, digestType, max
 };
 
 exports.interpolateName = function interpolateName(loaderContext, name, options) {
-	var filename = name || "[hash].[ext]";
+	var filename;
+	if (typeof name === 'function') {
```

Comment:
Sorry for nitpicking, but please use double quotes 😁  (you can push force on this branch)